.circleci/config.yml

```diff
@@ -8,71 +8,106 @@ jobs:
       - checkout
       - run: git submodule sync
       - run: git submodule update --init
-      - restore_cache:
```

Comment:
Not restoring cache from past builds, even if those don't completely match this build's `Cargo.lock`, will make every build job re-fetch every crate.

Reply:
thanks, I've put back the restore cache here too now